routing/user.js

```diff
@@ -0,0 +1,26 @@
+const User = require('../models/user');
+const bcrypt = require('bcrypt')
+
+const registerUser = async(req,res) =>{
+ try{
+  const saltRound = 10;
+  const hashpassword = await bcrypt.hash(req.body.password,saltRound);
+  const newUser = new User({
+    name:req.body.name,
```

Comment:
verify and check necessary fields before passing to database and return appropriate response

Reply:
> verify and check necessary fields before passing to database and return appropriate response
joi validation(middleware) ma error nakhse if required hasene nai provide kri hoi value